packages/web3torrent/package.json

```diff
@@ -9,26 +9,66 @@
     "not op_mini all"
   ],
   "dependencies": {
+    "@babel/core": "7.6.0",
     "@rimble/connection-banner": "1.2.3",
     "@statechannels/channel-client": "0.0.1",
     "@statechannels/devtools": "0.1.23",
+    "@svgr/webpack": "4.3.2",
+    "@typescript-eslint/eslint-plugin": "^2.2.0",
+    "@typescript-eslint/parser": "^2.2.0",
+    "babel-eslint": "10.0.3",
+    "babel-jest": "^24.9.0",
```

Comment:
Lots of these dependencies look like dev dependencies. (Are they even all used?)

Reply:
Yeah, I'm not sure why `CRA` decides to add them all to `dependencies` instead of `devDependencies`. We can probably clean these up as we go.

Reply:
https://www.npmjs.com/package/yarn-check might give some insight

Reply:
Unfortunately `yarn-check` doesn't seem to report correct information about `not used`. IE: It flags `typescript`, `rxjs`, `webtorrent` as not used.

Reply:
Also a lot of these dependencies (like `babel`) are baked into CRA so I think to remove them we need to update various things (like the `webpack.config`, jest configuration in `package.json`) etc..